config.yaml.dist

```diff
@@ -10,6 +10,7 @@
 loop:
     active: yes
     sleeping_time: 600
+    refresh_config: no
```

Comment:
Bit confused here. Doesn't setting 'refresh_config' to 'no' here actually enable it?

Reply:
Why do you think so? I just tested and it is not reloading the config when the value ist set to "no"